src/hasher.cpp

```diff
@@ -77,14 +49,6 @@ namespace libtorrent
 #elif TORRENT_USE_COMMONCRYPTO
 		CC_SHA1_Init(&m_context);
 #elif TORRENT_USE_CRYPTOAPI
```

Comment:
last #else contains fallback to builtin crypto routine, so this #elif can not be removed